### ISSUE TITLE
"invalid digit found in string" error

### ISSUE THREAD

**Issue by paperfrog95** [paritytech/substrate-api-sidecar#1088]
Hello, We are preparing a DOT coin wallet.

That's why we're collecting all the blocks.
And While collecting block `12532647`, We encountered the following error.

We solved this problem after updating the Sidecar version.
However, I couldn't find anything related to this in the release notes.
Was the update of version 14 mandatory?

We need to prevent a recurrence. Please check if the problem is really the version.

An error has never occurred in Testnet(Westend). (Testnet's sidecar version is v13.0.0)

## Environment
- Sidecar (v13.0.0) Error -> (v14.0.0) Solved
- Polkadot (v0.9.29)
```
{"code":500,"message":"invalid digit found in string","stack":"Error: invalid digit found in string\n    at 
module.exports.__wbindgen_error_new (/usr/src/app/node_modules/@substrate/calc/calc.js:260:17)\n    at 
<wasm_bindgen::JsError as core::convert::From<E>>::from::h82f95d9dbece96ef (wasm://wasm/0002745e:wasm-
function[40]:0x6047)\n    at calc_partial_fee (wasm://wasm/0002745e:wasm-function[7]:0x2829)\n    at 
module.exports.calc_partial_fee (/usr/src/app/node_modules/@substrate/calc/calc.js:181:14)\n    at BlocksService.calcPartialFee 
(/usr/src/app/build/src/services/blocks/BlocksService.js:237:48)\n    at BlocksService.fetchQueryFeeDetails 
(/usr/src/app/build/src/services/blocks/BlocksService.js:210:38)\n    at runMicrotasks (<anonymous>)\n    at 
processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async BlocksService.fetchBlock 
(/usr/src/app/build/src/services/blocks/BlocksService.js:161:35)\n    at async BlocksController.getBlockById 
(/usr/src/app/build/src/controllers/blocks/BlocksController.js:151:49)","level":"error"}`
```

**Comment by TarikGul** [comment]
So I believe the issue was with weights v1.5 which was fixed with the update of sidecar v13.1.0. That release specifically increased the versions of polkadot.js which dealt with the issue. 

As we moved from 1D to 2D weights there was an issue with the values being passed in which were not integers, since the value is now a struct that contains the correct value under `ref_time`.

This is a good note though, we could be a little more proactive with writing the exact issue that is being fixed in the release notes.
